src/pages/search.tsx

```diff
@@ -17,11 +18,20 @@ export default function PageGatsbySearch(props: PageProps): ReactElement {
     categories
   const title = owner
     ? `Published by ${accountTruncate(owner as string)}`
-    : `Search for ${searchValue || 'all data sets'}`
+    : `${
+        totalResults
+          ? searchValue
+            ? totalResults + ' results for ' + searchValue
```

Comment:
small change could be to also account for singular & plural, so the `s` needs to be removed if there's only 1 result. And if there's no result it could say `No results`

Reply:
sure, i'll do that